### DISCUSSION TITLE
loan eligibility prediction using java

### DISCUSSION THREAD

**Discussion by cgreddy2103** [jtablesaw/tablesaw#920] [Q&A]
What to do for data processing.I am not getting it. can you please tell what to do.

**Comment by lwhite1** [comment]
You have to be more specific.

The documentation is linked from the main page. It is outdated, but could
provide a starting point. You could then look to the tests for example code
that is up-to-date. Each subproject has its own test folder.

**Reply by cgreddy2103** [reply]
how to create dependent and independent variables in java

**Comment by cgreddy2103** [comment]
It is showing unsubscribe link isn't valid for your account

**Reply by benmccann** [reply]
I have no difficulty viewing the documentation: https://jtablesaw.github.io/tablesaw/

**Reply by cgreddy2103** [reply]
How to fill the missing values of a column using mode function in java

**Reply by lwhite1** [reply]
You have to compute the mode. I would start with the method `table.countBy("column")`;
 
you can set the missing values using
myColumn.setMissing(theMode);

**Reply by cgreddy2103** [reply]
What does mycolumn, theMode represent in the above code.

**Reply by lwhite1** [reply]
They're imaginary variable names. 

Is this a homework assignment? 
In any case, we're happy to help with questions related to tablesaw, but can't really teach you to program or do all your work for you.
